src/test/system_tests/ceph_s3_tests/test_ceph_s3.js

```diff
@@ -28,7 +28,7 @@ let tests_list;
 
 const OUT_OF_SCOPE_TESTS = create_out_of_scope_tests_list() || [];
 //Regexp match will be tested per each entry
-const S3_CEPH_TEST_OUT_OF_SCOPE_REGEXP = new RegExp(`(${OUT_OF_SCOPE_TESTS.join(')|(')})`);
+const S3_CEPH_TEST_OUT_OF_SCOPE_REGEXP = new RegExp(`(${OUT_OF_SCOPE_TESTS.join('\\b)|(')}\\b)`);
```

Comment:
Would you please explain the change?

Reply:
when creating the regex from the out of scope tests lists (in our case black list and pending list), I add /b at the end of each test name. that tell the regex to match the test names to the end of the string. so if the test has any more characters after the test name (for example test_lifecycle_set_date compared to test_lifecycle_set) it will not match. 

I can add /b to the beginning as  well to make to match it to the beginning of the string as well. but I though that would be redundant since all the tests starts the same anyway